### ISSUE TITLE
psh: ls -l: year is not printed for files with non current year dates

### ISSUE THREAD

**Issue by damianloew** [phoenix-rtos/phoenix-rtos-project#552]
Current behavior:
![Screenshot from 2022-11-18 18-46-25](https://user-images.githubusercontent.com/77304431/202769157-33408b9c-c43f-4114-9e93-61d4bae7aa7c.png)
Intended behavior:
![Screenshot from 2022-11-18 18-45-02](https://user-images.githubusercontent.com/77304431/202769191-db7c52d4-95ec-4100-b22b-409539763b0a.png)

**Comment by jmaksymowicz** [comment]
I haven't noticed the problem with seconds being printed instead of year. The current version of `ls -l` command never prints the year. The original title of this issue is correct though, other implementations of `ls` do this so it would be useful to have this feature.

**Comment by nalajcie** [comment]
Haven't tested the current code but until recently this still was a problem (DTR project 1.07.04, jffs2 rootfs):

![image](https://github.com/phoenix-rtos/phoenix-rtos-project/assets/3939983/074642c5-40e7-4a07-af65-6e86d15a7990)

**Comment by damianloew** [comment]
> I haven't noticed the problem with seconds being printed instead of year. The current version of `ls -l` command never prints the year. The original title of this issue is correct though, other implementations of `ls` do this so it would be useful to have this feature.

Yeah, that's the point - it would be useful to print year if it's different than the current one.
